### PR TITLE
fix: adjust edit blockchain label

### PR DESCRIPTION
**Pull request by lukicenturi**
Need https://github.com/rotki/rotki/pull/8627 to be merged first

### CODE REVIEW

File: frontend/app/src/composables/api/blockchain/accounts.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ function payloadToData({ address, label, tags }: Omit<BlockchainAccountPayload,
     accounts: [
       {
         address,
-        label: label || null,
+        label: isDefined(label) ? label : null,
```
Comment (kelsos) [inline frontend/app/src/composables/api/blockchain/accounts.ts:48]: wouldn' that also allow '' as a valid label?

Comment (lukicenturi) [reply frontend/app/src/composables/api/blockchain/accounts.ts:48]: yes, that's a way to tell backend to remove it from addressbook

File: frontend/app/src/components/accounts/management/types/AgnosticAddressAccountForm.vue [hunk 2]
```diff
@@ -40,12 +39,11 @@ const label = computed<string>({
...
   set(label: string) {
     const model = get(modelValue);
-    const labelData = label ? { label } : {};
     set(modelValue, {
       ...model,
       data: {
-        ...objectOmit(model.data, ['label']),
-        ...labelData,
+        ...model.data,
+        label,
```
Comment (kelsos) [inline frontend/app/src/components/accounts/management/types/AgnosticAddressAccountForm.vue:46]: this I guess needs the changes to work right? and we are sending the label either way?
Do we send the `` label too?

**Review by kelsos — APPROVED** [review]
lgtm thanks

**Pull request merged**